atd-vze/src/views/Crashes/Maps/CrashEditCoordsMap.js

```diff
@@ -159,7 +164,11 @@ class CrashEditCoordsMap extends Component {
           ref={this.mapRef}
           width="100%"
           height="350px"
-          mapStyle={`mapbox://styles/mapbox/${mapStyle}-v9`}
+          mapStyle={
+            isDev
+              ? "mapbox://styles/mapbox/satellite-streets-v11"
+              : LOCATION_MAP_CONFIG.mapStyle
+          }
```

Comment:
Lifted directly from @johnclary's work - thank you for that!

Reply:
pretty sure i copied Chia...